docs/admin/admin_install.md

```diff
@@ -93,14 +95,16 @@ The plugin behavior can be controlled with the following list of settings.
 | enable_sotagg             | True                          | True    | A boolean to represent whether or not to provide a GraphQL query per device to allow the intended configuration to provide data variables to the plugin.                   |
-| enable_postprocessing     | True                          | False    | A boolean to represent whether or not to generate intended configurations to push, with extra processing such as secrets rendering.                                        |
+| enable_postprocessing     | True                          | False    | A boolean to represent whether or not to generate intended configurations to push, with extra processing such as secrets rendering.                                       |
 | postprocessing_callables  | ['mypackage.myfunction']      | []      | A list of function paths, in dotted format, that are appended to the available methods for post-processing the intended configuration, for instance, the `render_secrets`. |
 | postprocessing_subscribed | ['mypackage.myfunction']      | []      | A list of function paths, that should exist as postprocessing_callables, that defines the order of application of during the post-processing process.                      |
-| platform_slug_map         | {"cisco_wlc": "cisco_aireos"} | None    | A dictionary in which the key is the platform slug and the value is what netutils uses in any "network_os" parameter within `netutils.config.compliance.parser_map`.                                                      |
+| platform_slug_map         | {"cisco_wlc": "cisco_aireos"} | None    | A dictionary in which the key is the platform slug and the value is what netutils uses in any "network_os" parameter within `netutils.config.compliance.parser_map`.       |
 | sot_agg_transposer        | "mypkg.transposer"            | None    | A string representation of a function that can post-process the graphQL data.                                                                                              |
 | per_feature_bar_width     | 0.15                          | 0.15    | The width of the table bar within the overview report                                                                                                                      |
 | per_feature_width         | 13                            | 13      | The width in inches that the overview table can be.                                                                                                                        |
 | per_feature_height        | 4                             | 4       | The height in inches that the overview table can be.                                                                                                                       |
+| jinja_env_trim_blocks     | True                          | True    | A boolean to represent whether the jinja2 option for "trim_blocks" should be enabled for intended config rendering                                                         |
```

Comment:
After getting internal feedback, I think the point @jeffkala and @electro47h is that there are many variables one could set for Jinja, I think the list is:

- block_start_string
- block_end_string
- variable_start_string
- variable_end_string
- comment_start_string
- comment_end_string
- line_statement_prefix
- line_comment_prefix
- trim_blocks
- lstrip_blocks
- newline_sequence
- keep_trailing_newline
- extensions
- optimized
- undefined
- finalize
- autoescape
- loader
- cache_size
- auto_reload
- bytecode_cache
- enable_async

It probably makes sense to support them, as a dictionary, with the 3 variables you have set as defaults. So, there should likely be one variable called `jinja_env` that is set. If you want to have a magic method of some sort such that anything set with `jinja_env_*` automagically gets into the dictionary, I could buy that as well, as long as it is in addition to the dictionary method.

Reply:
Understood. None of those were possible to set before, but since they will be now, they should be included. This will take a good bit of rework on my end. I'm going to draft this PR and work on it later this week if time allows.

Reply:
Question for the masses here. For an option such as 'undefined', how would that look in the nautobot_config.py file for a setting?

```
from jinja2 import StrictUndefined

Environment(undefined=StrictUndefined, lstrip_blocks=...)
```

It appears most of the settings are boolean or string values, but a few of the options listed for Jinja are enumerations or classes. If the user intended to customize that setting in their nautobot_config.py file, would they have to add the import, or is there another way to represent that?

Reply:
> If the user intended to customize that setting in their nautobot_config.py file, would they have to add the import

Yes, they would have to add that to the import within nautobot_config.py. We could work around it, but the complexity is not really worth it.

Reply:
Thank you. I have committed what my interpretation of the request is.

I'm out of the office tomorrow and will be unable to test. We also have to get our environment upgraded from 1.5.24 to >=1.6.1 to install this version of the plugin. I will work to get this tested when I can.

Please let me know if this is the correct architecture that you're looking for, or if I need to make changes. Happy to keep plugging away at this.